profiles/common/modules/custom/tmgmt_poetry/tmgmt_poetry.module

```diff
@@ -205,6 +205,15 @@ function _tmgmt_poetry_form_load_translator_settings(&$form_state, &$form, $job)
     $settings = [];
   }
 
+  // NEPT-62: Overwriting the 'delai' field properties.
+  if (isset($settings['delai'])) {
+    $settings['delai']['#title'] = t('Requested delivery date');
+    $settings['delai']['#date_year_range'] = '-0:+1';
+    $settings['delai']['#default_value'] = '';
+    $settings['delai']['#required'] = TRUE;
+    $settings['delai']['#weight'] = -10;
```

Comment:
the ticket was not about making it in the top with setting up the #weight (it doesn't matter that much, but still the change makes more than it's expected!)